cmd/notation/internal/plugin/plugin.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"context"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/plugin"
+	"github.com/notaryproject/notation-go/plugin/proto"
+)
+
+// GetPluginMetadataIfExist returns plugin's metadata if it exists in Notation
+func GetPluginMetadataIfExist(ctx context.Context, pluginName string) (*proto.GetMetadataResponse, error) {
```

Comment:
@priteshbandi It is also used by `notation plugin install` command, but since this PR is only about `uninstall`, it appears that it's only used by `uninstall`. So yeah, it's a shared function.

Reply:
@priteshbandi Since I refactored code in `notation plugin install`, it no longer needs `GetPluginMetadataIfExist`. Therefore, as you suggested, I moved this function to `uninstall.go` and made it an unexport function.